addons/disable-sprite-wobble/addon.json

```diff
@@ -1,14 +1,10 @@
 {
-  "name": "Disable sprite wobble in sprite pane",
-  "description": "Allow scripts, costumes, and sounds to be moved to different sprites without the sprites moving around within the sprite pane.",
+  "name": "Fix sprite pane glitching",
+  "description": "Allow scripts, costumes, and sounds to be dragged and dropped onto different sprites without the sprites jumping around within the sprite pane.",
   "credits": [
     {
-      "name": "Xan",
-      "link": "https://scratch.mit.edu/users/-Xanimation-/"
-    },
-    {
-      "name": "CST1229",
-      "link": "https://github.com/cst1229/"
+      "name": "DNin01",
+      "link": "https://github.com/DNin01"
     }
```

Comment:
I removed the previous contributors from this addon's credits because I completely redid the code (as if this was a new addon). Is that okay?

BTW, I also used [mxmou's code](https://github.com/LLK/scratch-gui/issues/5471#issuecomment-1345238522).

Reply:
I think I'm leaning towards leaving them, since they still had an effect on the addon.

Reply:
I guess all the credits should stay because we're keeping the option to remove the wobble.